src/lib/api/organizations.ts

```diff
@@ -45,10 +42,14 @@ export const OrganizationAPI = {
    */
   create: async (organization: Organization): Promise<Organization> =>
     EntityApi.createFn<Organization, Organization>(
-      `${siteConfig.env.backendServiceURL}/organizations`,
+      ORGANIZATIONS_BASEURL,
       organization
     ),
 
+  createNested: async (id: Id, entity: Organization): Promise<Organization> => {
```

Comment:
Thoughts on having the same API for all entities, but some of them currently don't (or never will make sense to) implement certain functions? This is about having a standard and predictable set of entity verbs.

Reply:
Yeah - because it's a means to achieve standardization this seems fine.